sklearn/cross_validation.py

```diff
@@ -213,8 +213,8 @@ def __repr__(self):
         )
 
     def __len__(self):
-        return int(factorial(self.n) / factorial(self.n - self.p)
-                   / factorial(self.p))
+        return int(factorial(self.n) / factorial(self.n - self.p) /
```

Comment:
Ha! I can't believe we're actually calculating these factorials when scipy provides a [binomial coefficient function](http://docs.scipy.org/doc/scipy/reference/generated/scipy.misc.comb.html#scipy.misc.comb).